.github/workflows/build.yml

```diff
@@ -285,6 +285,10 @@ jobs:
 
         commit_titles | TERM=xterm-color .github/scripts/check-commit-titles.sh
 
+    - name: Check DCO
+      uses: christophebedard/dco-check@0.5.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We should exclude dependabot from the check (eg using `--exclude-emails '49699333+dependabot[bot]@users.noreply.github.com'`)

Reply:
done ;)